projects/plugins/protect/src/js/components/interstitial/index.jsx

```diff
@@ -54,7 +54,9 @@ const SecurityBundle = ( { onAdd, redirecting, ...rest } ) => {
  */
 const Interstitial = ( { onSecurityAdd, securityJustAdded } ) => {
 	const [ isMediumSize ] = useBreakpointMatch( 'md' );
-	const mediaClassName = isMediumSize ? styles[ 'is-viewport-medium' ] : null;
+	const mediaClassName = `${ styles.section } ${
+		isMediumSize ? styles[ 'is-viewport-medium' ] : null
```

Comment:
```suggestion
		isMediumSize ? styles[ 'is-viewport-medium' ] : ''
```

I believe that using `null` in the template string will result in a literal "null" class name potentially being applied, if you would like to keep the ternary here we could just use an empty string `''` instead.